src/Components/ActivationKeys/__tests__/ActivationKeys.test.js

```diff
@@ -118,4 +137,13 @@ describe('ActivationKeys', () => {
       expect(container).toMatchSnapshot();
     });
   });
+
+  describe('show blank state when no activation keys', () => {
+    def('keysData', () => []);
+
+    it('renders blank state', async () => {
+      const { container } = render(<PageContainer />);
+      expect(container).toMatchSnapshot();
```

Comment:
I think explicit component test is better than snapshot testing. But this is minor, feel free to ignore if yo u think different.